From: Benjamin Drung <benjamin.drung@ionos.com>
Date: Tue, 16 Nov 2021 15:40:43 +0100
Subject: Do not require contextvars for Python >= 3.7

Python >= 3.7 comes with an `contextvars` module and therefore a
separate `contextvars` package is not needed. Otherwise salt will
needlessly fail:

pkg_resources.DistributionNotFound: The 'contextvars' distribution was
not found and is required by salt

Bug-Debian: https://bugs.debian.org/999590
Forwarded: https://github.com/saltstack/salt/pull/61240
Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>
---
 requirements/base.txt                      | 2 +-
 requirements/static/ci/py3.10/cloud.txt    | 4 ----
 requirements/static/ci/py3.10/darwin.txt   | 4 ----
 requirements/static/ci/py3.10/docs.txt     | 4 ----
 requirements/static/ci/py3.10/freebsd.txt  | 4 ----
 requirements/static/ci/py3.10/lint.txt     | 4 ----
 requirements/static/ci/py3.10/linux.txt    | 4 ----
 requirements/static/ci/py3.10/windows.txt  | 6 ------
 requirements/static/ci/py3.5/linux.txt     | 2 +-
 requirements/static/ci/py3.6/cloud.txt     | 2 +-
 requirements/static/ci/py3.6/docs.txt      | 2 +-
 requirements/static/ci/py3.6/lint.txt      | 2 +-
 requirements/static/ci/py3.6/linux.txt     | 2 +-
 requirements/static/ci/py3.7/cloud.txt     | 5 -----
 requirements/static/ci/py3.7/darwin.txt    | 4 ----
 requirements/static/ci/py3.7/docs.txt      | 4 ----
 requirements/static/ci/py3.7/freebsd.txt   | 4 ----
 requirements/static/ci/py3.7/lint.txt      | 5 -----
 requirements/static/ci/py3.7/linux.txt     | 4 ----
 requirements/static/ci/py3.7/windows.txt   | 4 ----
 requirements/static/ci/py3.8/cloud.txt     | 4 ----
 requirements/static/ci/py3.8/darwin.txt    | 4 ----
 requirements/static/ci/py3.8/docs.txt      | 4 ----
 requirements/static/ci/py3.8/freebsd.txt   | 4 ----
 requirements/static/ci/py3.8/lint.txt      | 4 ----
 requirements/static/ci/py3.8/linux.txt     | 4 ----
 requirements/static/ci/py3.8/windows.txt   | 4 ----
 requirements/static/ci/py3.9/cloud.txt     | 4 ----
 requirements/static/ci/py3.9/darwin.txt    | 4 ----
 requirements/static/ci/py3.9/docs.txt      | 4 ----
 requirements/static/ci/py3.9/freebsd.txt   | 4 ----
 requirements/static/ci/py3.9/lint.txt      | 4 ----
 requirements/static/ci/py3.9/linux.txt     | 4 ----
 requirements/static/ci/py3.9/windows.txt   | 4 ----
 requirements/static/pkg/py3.10/darwin.txt  | 4 ----
 requirements/static/pkg/py3.10/freebsd.txt | 4 ----
 requirements/static/pkg/py3.10/linux.txt   | 4 ----
 requirements/static/pkg/py3.10/windows.txt | 4 ----
 requirements/static/pkg/py3.5/linux.txt    | 2 +-
 requirements/static/pkg/py3.6/linux.txt    | 2 +-
 requirements/static/pkg/py3.7/darwin.txt   | 4 ----
 requirements/static/pkg/py3.7/freebsd.txt  | 4 ----
 requirements/static/pkg/py3.7/linux.txt    | 4 ----
 requirements/static/pkg/py3.7/windows.txt  | 4 ----
 requirements/static/pkg/py3.8/darwin.txt   | 4 ----
 requirements/static/pkg/py3.8/freebsd.txt  | 4 ----
 requirements/static/pkg/py3.8/linux.txt    | 4 ----
 requirements/static/pkg/py3.8/windows.txt  | 4 ----
 requirements/static/pkg/py3.9/darwin.txt   | 4 ----
 requirements/static/pkg/py3.9/freebsd.txt  | 4 ----
 requirements/static/pkg/py3.9/linux.txt    | 4 ----
 requirements/static/pkg/py3.9/windows.txt  | 4 ----
 52 files changed, 8 insertions(+), 188 deletions(-)

diff --git a/requirements/base.txt b/requirements/base.txt
index 1fd7507..ab57338 100644
--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -4,5 +4,5 @@ PyYAML
 MarkupSafe
 requests>=1.0.0
 distro>=1.0.1
-contextvars
+contextvars ; python_version < "3.7"
 psutil>=5.0.0
diff --git a/requirements/static/ci/py3.10/cloud.txt b/requirements/static/ci/py3.10/cloud.txt
index 5766696..8fd49d1 100644
--- a/requirements/static/ci/py3.10/cloud.txt
+++ b/requirements/static/ci/py3.10/cloud.txt
@@ -372,8 +372,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -419,8 +417,6 @@ google-auth==2.1.0
     # via kubernetes
 idna==2.8
     # via requests
-immutables==0.16
-    # via contextvars
 iniconfig==1.1.1
     # via pytest
 ipaddress==1.0.23
diff --git a/requirements/static/ci/py3.10/darwin.txt b/requirements/static/ci/py3.10/darwin.txt
index f93aac8..40f73f8 100644
--- a/requirements/static/ci/py3.10/darwin.txt
+++ b/requirements/static/ci/py3.10/darwin.txt
@@ -384,8 +384,6 @@ clustershell==1.8.1
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -448,8 +446,6 @@ idna==2.8
     #   -r requirements/darwin.txt
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 iniconfig==1.0.1
     # via pytest
 ipaddress==1.0.22
diff --git a/requirements/static/ci/py3.10/docs.txt b/requirements/static/ci/py3.10/docs.txt
index 119c6f2..5133266 100644
--- a/requirements/static/ci/py3.10/docs.txt
+++ b/requirements/static/ci/py3.10/docs.txt
@@ -385,8 +385,6 @@ colorama==0.4.4
     # via ciscoconfparse
 contextlib2==0.5.5
     # via cherrypy
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -433,8 +431,6 @@ idna==2.8
     # via requests
 imagesize==1.2.0
     # via sphinx
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4
     # via moto
 ipaddress==1.0.23
diff --git a/requirements/static/ci/py3.10/freebsd.txt b/requirements/static/ci/py3.10/freebsd.txt
index c4d8209..073466d 100644
--- a/requirements/static/ci/py3.10/freebsd.txt
+++ b/requirements/static/ci/py3.10/freebsd.txt
@@ -382,8 +382,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -444,8 +442,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 iniconfig==1.0.1
     # via pytest
 ipaddress==1.0.22
diff --git a/requirements/static/ci/py3.10/lint.txt b/requirements/static/ci/py3.10/lint.txt
index ef3c860..2c4828f 100644
--- a/requirements/static/ci/py3.10/lint.txt
+++ b/requirements/static/ci/py3.10/lint.txt
@@ -379,8 +379,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.4.7
@@ -423,8 +421,6 @@ google-auth==2.0.1
     # via kubernetes
 idna==3.2
     # via requests
-immutables==0.16
-    # via contextvars
 importlib-metadata==4.6.4
     # via moto
 ipaddress==1.0.23
diff --git a/requirements/static/ci/py3.10/linux.txt b/requirements/static/ci/py3.10/linux.txt
index cf4a5a0..b5aba9f 100644
--- a/requirements/static/ci/py3.10/linux.txt
+++ b/requirements/static/ci/py3.10/linux.txt
@@ -396,8 +396,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -457,8 +455,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 iniconfig==1.0.1
     # via pytest
 ipaddress==1.0.22
diff --git a/requirements/static/ci/py3.10/windows.txt b/requirements/static/ci/py3.10/windows.txt
index ddd9217..da2a769 100644
--- a/requirements/static/ci/py3.10/windows.txt
+++ b/requirements/static/ci/py3.10/windows.txt
@@ -71,8 +71,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.1
     # via pytest
-contextvars==2.4
-    # via -r requirements/static/pkg/py3.10/windows.txt
 cryptography==3.4.7
     # via
     #   -r requirements/static/pkg/py3.10/windows.txt
@@ -129,10 +127,6 @@ idna==2.8
     #   -r requirements/static/pkg/py3.10/windows.txt
     #   moto
     #   requests
-immutables==0.15
-    # via
-    #   -r requirements/static/pkg/py3.10/windows.txt
-    #   contextvars
 iniconfig==1.0.1
     # via pytest
 ioloop==0.1a0
diff --git a/requirements/static/ci/py3.5/linux.txt b/requirements/static/ci/py3.5/linux.txt
index b1c583d..033f492 100644
--- a/requirements/static/ci/py3.5/linux.txt
+++ b/requirements/static/ci/py3.5/linux.txt
@@ -387,7 +387,7 @@ click==7.1.1
     # via geomet
 clustershell==1.8.3
     # via -r requirements/static/ci/common.in
-contextvars==2.4
+contextvars==2.4 ; python_version < "3.7"
     # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
diff --git a/requirements/static/ci/py3.6/cloud.txt b/requirements/static/ci/py3.6/cloud.txt
index 1deef84..61171d4 100644
--- a/requirements/static/ci/py3.6/cloud.txt
+++ b/requirements/static/ci/py3.6/cloud.txt
@@ -369,7 +369,7 @@ click==8.0.1
     # via geomet
 clustershell==1.8.3
     # via -r requirements/static/ci/common.in
-contextvars==2.4
+contextvars==2.4 ; python_version < "3.7"
     # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
diff --git a/requirements/static/ci/py3.6/docs.txt b/requirements/static/ci/py3.6/docs.txt
index 606bbb1..a78e4b8 100644
--- a/requirements/static/ci/py3.6/docs.txt
+++ b/requirements/static/ci/py3.6/docs.txt
@@ -390,7 +390,7 @@ click==8.0.1
     # via geomet
 clustershell==1.8.3
     # via -r requirements/static/ci/common.in
-contextvars==2.4
+contextvars==2.4 ; python_version < "3.7"
     # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
diff --git a/requirements/static/ci/py3.6/lint.txt b/requirements/static/ci/py3.6/lint.txt
index 6a97cee..8d9db9d 100644
--- a/requirements/static/ci/py3.6/lint.txt
+++ b/requirements/static/ci/py3.6/lint.txt
@@ -378,7 +378,7 @@ click==8.0.1
     # via geomet
 clustershell==1.8.3
     # via -r requirements/static/ci/common.in
-contextvars==2.4
+contextvars==2.4 ; python_version < "3.7"
     # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
diff --git a/requirements/static/ci/py3.6/linux.txt b/requirements/static/ci/py3.6/linux.txt
index be533a4..7f19949 100644
--- a/requirements/static/ci/py3.6/linux.txt
+++ b/requirements/static/ci/py3.6/linux.txt
@@ -391,7 +391,7 @@ click==7.1.1
     # via geomet
 clustershell==1.8.3
     # via -r requirements/static/ci/common.in
-contextvars==2.4
+contextvars==2.4 ; python_version < "3.7"
     # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
diff --git a/requirements/static/ci/py3.7/cloud.txt b/requirements/static/ci/py3.7/cloud.txt
index ef8f447..a0ecb0b 100644
--- a/requirements/static/ci/py3.7/cloud.txt
+++ b/requirements/static/ci/py3.7/cloud.txt
@@ -372,8 +372,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -419,8 +417,6 @@ google-auth==2.1.0
     # via kubernetes
 idna==2.8
     # via requests
-immutables==0.16
-    # via contextvars
 importlib-metadata==4.8.1 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
@@ -841,7 +837,6 @@ transitions==0.8.9
 typing-extensions==3.10.0.0
     # via
     #   gitpython
-    #   immutables
     #   importlib-metadata
 urllib3==1.26.6
     # via
diff --git a/requirements/static/ci/py3.7/darwin.txt b/requirements/static/ci/py3.7/darwin.txt
index 75f0d9d..970447d 100644
--- a/requirements/static/ci/py3.7/darwin.txt
+++ b/requirements/static/ci/py3.7/darwin.txt
@@ -384,8 +384,6 @@ clustershell==1.8.1
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -448,8 +446,6 @@ idna==2.8
     #   -r requirements/darwin.txt
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/darwin.txt
diff --git a/requirements/static/ci/py3.7/docs.txt b/requirements/static/ci/py3.7/docs.txt
index a428e97..a51c3da 100644
--- a/requirements/static/ci/py3.7/docs.txt
+++ b/requirements/static/ci/py3.7/docs.txt
@@ -393,8 +393,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.4.7
@@ -453,8 +451,6 @@ idna==2.8
     #   requests
 imagesize==1.2.0
     # via sphinx
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
diff --git a/requirements/static/ci/py3.7/freebsd.txt b/requirements/static/ci/py3.7/freebsd.txt
index 1be5c42..ca2e428 100644
--- a/requirements/static/ci/py3.7/freebsd.txt
+++ b/requirements/static/ci/py3.7/freebsd.txt
@@ -382,8 +382,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -444,8 +442,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/freebsd.in
diff --git a/requirements/static/ci/py3.7/lint.txt b/requirements/static/ci/py3.7/lint.txt
index 6008677..09dd78b 100644
--- a/requirements/static/ci/py3.7/lint.txt
+++ b/requirements/static/ci/py3.7/lint.txt
@@ -381,8 +381,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.4.7
@@ -425,8 +423,6 @@ google-auth==2.0.1
     # via kubernetes
 idna==3.2
     # via requests
-immutables==0.16
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
@@ -831,7 +827,6 @@ typed-ast==1.4.1
 typing-extensions==3.10.0.0
     # via
     #   gitpython
-    #   immutables
     #   importlib-metadata
 tzlocal==3.0
     # via apscheduler
diff --git a/requirements/static/ci/py3.7/linux.txt b/requirements/static/ci/py3.7/linux.txt
index fa33ef5..992779f 100644
--- a/requirements/static/ci/py3.7/linux.txt
+++ b/requirements/static/ci/py3.7/linux.txt
@@ -394,8 +394,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -455,8 +453,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
diff --git a/requirements/static/ci/py3.7/windows.txt b/requirements/static/ci/py3.7/windows.txt
index 4bd4114..c284a1e 100644
--- a/requirements/static/ci/py3.7/windows.txt
+++ b/requirements/static/ci/py3.7/windows.txt
@@ -67,8 +67,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.1
     # via pytest
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.4.7
     # via
     #   -r requirements/windows.txt
@@ -120,8 +118,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/windows.txt
diff --git a/requirements/static/ci/py3.8/cloud.txt b/requirements/static/ci/py3.8/cloud.txt
index 44f69a3..3f40c68 100644
--- a/requirements/static/ci/py3.8/cloud.txt
+++ b/requirements/static/ci/py3.8/cloud.txt
@@ -372,8 +372,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -419,8 +417,6 @@ google-auth==2.1.0
     # via kubernetes
 idna==2.8
     # via requests
-immutables==0.16
-    # via contextvars
 importlib-metadata==4.8.1 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/linux.in
 iniconfig==1.1.1
diff --git a/requirements/static/ci/py3.8/darwin.txt b/requirements/static/ci/py3.8/darwin.txt
index 0806e24..f16fe97 100644
--- a/requirements/static/ci/py3.8/darwin.txt
+++ b/requirements/static/ci/py3.8/darwin.txt
@@ -384,8 +384,6 @@ clustershell==1.8.1
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -448,8 +446,6 @@ idna==2.8
     #   -r requirements/darwin.txt
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/darwin.txt
 iniconfig==1.0.1
diff --git a/requirements/static/ci/py3.8/docs.txt b/requirements/static/ci/py3.8/docs.txt
index 8ed0711..8822e26 100644
--- a/requirements/static/ci/py3.8/docs.txt
+++ b/requirements/static/ci/py3.8/docs.txt
@@ -383,8 +383,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -431,8 +429,6 @@ idna==2.8
     # via requests
 imagesize==1.2.0
     # via sphinx
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
diff --git a/requirements/static/ci/py3.8/freebsd.txt b/requirements/static/ci/py3.8/freebsd.txt
index 5adbb2d..d1a1b04 100644
--- a/requirements/static/ci/py3.8/freebsd.txt
+++ b/requirements/static/ci/py3.8/freebsd.txt
@@ -382,8 +382,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -444,8 +442,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/freebsd.in
 iniconfig==1.0.1
diff --git a/requirements/static/ci/py3.8/lint.txt b/requirements/static/ci/py3.8/lint.txt
index dd4a365..dfae3de 100644
--- a/requirements/static/ci/py3.8/lint.txt
+++ b/requirements/static/ci/py3.8/lint.txt
@@ -381,8 +381,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.4.7
@@ -425,8 +423,6 @@ google-auth==2.0.1
     # via kubernetes
 idna==3.2
     # via requests
-immutables==0.16
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
diff --git a/requirements/static/ci/py3.8/linux.txt b/requirements/static/ci/py3.8/linux.txt
index 13bc9e9..88f6247 100644
--- a/requirements/static/ci/py3.8/linux.txt
+++ b/requirements/static/ci/py3.8/linux.txt
@@ -394,8 +394,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -455,8 +453,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/linux.in
 iniconfig==1.0.1
diff --git a/requirements/static/ci/py3.8/windows.txt b/requirements/static/ci/py3.8/windows.txt
index fb616b3..408bea1 100644
--- a/requirements/static/ci/py3.8/windows.txt
+++ b/requirements/static/ci/py3.8/windows.txt
@@ -67,8 +67,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.1
     # via pytest
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.4.7
     # via
     #   -r requirements/windows.txt
@@ -120,8 +118,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/windows.txt
 iniconfig==1.0.1
diff --git a/requirements/static/ci/py3.9/cloud.txt b/requirements/static/ci/py3.9/cloud.txt
index 991bf7c..e166a96 100644
--- a/requirements/static/ci/py3.9/cloud.txt
+++ b/requirements/static/ci/py3.9/cloud.txt
@@ -372,8 +372,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -419,8 +417,6 @@ google-auth==2.1.0
     # via kubernetes
 idna==2.8
     # via requests
-immutables==0.16
-    # via contextvars
 importlib-metadata==4.8.1 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/linux.in
 iniconfig==1.1.1
diff --git a/requirements/static/ci/py3.9/darwin.txt b/requirements/static/ci/py3.9/darwin.txt
index 75fbad6..1f445a7 100644
--- a/requirements/static/ci/py3.9/darwin.txt
+++ b/requirements/static/ci/py3.9/darwin.txt
@@ -384,8 +384,6 @@ clustershell==1.8.1
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -448,8 +446,6 @@ idna==2.8
     #   -r requirements/darwin.txt
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/darwin.txt
 iniconfig==1.0.1
diff --git a/requirements/static/ci/py3.9/docs.txt b/requirements/static/ci/py3.9/docs.txt
index 3079853..21069d3 100644
--- a/requirements/static/ci/py3.9/docs.txt
+++ b/requirements/static/ci/py3.9/docs.txt
@@ -381,8 +381,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -429,8 +427,6 @@ idna==2.8
     # via requests
 imagesize==1.2.0
     # via sphinx
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
diff --git a/requirements/static/ci/py3.9/freebsd.txt b/requirements/static/ci/py3.9/freebsd.txt
index 3931449..7f8cadb 100644
--- a/requirements/static/ci/py3.9/freebsd.txt
+++ b/requirements/static/ci/py3.9/freebsd.txt
@@ -382,8 +382,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -444,8 +442,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/freebsd.in
 iniconfig==1.0.1
diff --git a/requirements/static/ci/py3.9/lint.txt b/requirements/static/ci/py3.9/lint.txt
index 7f33eb9..d4873a3 100644
--- a/requirements/static/ci/py3.9/lint.txt
+++ b/requirements/static/ci/py3.9/lint.txt
@@ -379,8 +379,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.4
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==1.0.15 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.4.7
@@ -423,8 +421,6 @@ google-auth==2.0.1
     # via kubernetes
 idna==3.2
     # via requests
-immutables==0.16
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via
     #   -r requirements/static/pkg/linux.in
diff --git a/requirements/static/ci/py3.9/linux.txt b/requirements/static/ci/py3.9/linux.txt
index 5f81317..7d914d3 100644
--- a/requirements/static/ci/py3.9/linux.txt
+++ b/requirements/static/ci/py3.9/linux.txt
@@ -396,8 +396,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.3
     # via ciscoconfparse
-contextvars==2.4
-    # via -r requirements/base.txt
 croniter==0.3.29 ; sys_platform != "win32"
     # via -r requirements/static/ci/common.in
 cryptography==3.3.2
@@ -457,8 +455,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/linux.in
 iniconfig==1.0.1
diff --git a/requirements/static/ci/py3.9/windows.txt b/requirements/static/ci/py3.9/windows.txt
index ed8f8e7..764203e 100644
--- a/requirements/static/ci/py3.9/windows.txt
+++ b/requirements/static/ci/py3.9/windows.txt
@@ -67,8 +67,6 @@ clustershell==1.8.3
     # via -r requirements/static/ci/common.in
 colorama==0.4.1
     # via pytest
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.4.7
     # via
     #   -r requirements/windows.txt
@@ -120,8 +118,6 @@ idna==2.8
     # via
     #   moto
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/windows.txt
 iniconfig==1.0.1
diff --git a/requirements/static/pkg/py3.10/darwin.txt b/requirements/static/pkg/py3.10/darwin.txt
index a82118b..8cbf1a2 100644
--- a/requirements/static/pkg/py3.10/darwin.txt
+++ b/requirements/static/pkg/py3.10/darwin.txt
@@ -16,8 +16,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/darwin.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via
     #   -r requirements/darwin.txt
@@ -32,8 +30,6 @@ idna==2.8
     # via
     #   -r requirements/darwin.txt
     #   requests
-immutables==0.15
-    # via contextvars
 jaraco.classes==3.2.1
     # via jaraco.collections
 jaraco.collections==3.4.0
diff --git a/requirements/static/pkg/py3.10/freebsd.txt b/requirements/static/pkg/py3.10/freebsd.txt
index a6f2329..c2329a5 100644
--- a/requirements/static/pkg/py3.10/freebsd.txt
+++ b/requirements/static/pkg/py3.10/freebsd.txt
@@ -14,8 +14,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/freebsd.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
@@ -24,8 +22,6 @@ distro==1.5.0
     #   -r requirements/static/pkg/freebsd.in
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 jaraco.classes==3.2.1
     # via jaraco.collections
 jaraco.collections==3.4.0
diff --git a/requirements/static/pkg/py3.10/linux.txt b/requirements/static/pkg/py3.10/linux.txt
index 3e6269a..e69821f 100644
--- a/requirements/static/pkg/py3.10/linux.txt
+++ b/requirements/static/pkg/py3.10/linux.txt
@@ -14,16 +14,12 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/linux.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
     # via -r requirements/base.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 jaraco.classes==3.2.1
     # via jaraco.collections
 jaraco.collections==3.4.0
diff --git a/requirements/static/pkg/py3.10/windows.txt b/requirements/static/pkg/py3.10/windows.txt
index 1b006fc..9aea556 100644
--- a/requirements/static/pkg/py3.10/windows.txt
+++ b/requirements/static/pkg/py3.10/windows.txt
@@ -18,8 +18,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/windows.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.4.7
     # via
     #   -r requirements/windows.txt
@@ -32,8 +30,6 @@ gitpython==3.1.14
     # via -r requirements/windows.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 ioloop==0.1a0
     # via -r requirements/windows.txt
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.5/linux.txt b/requirements/static/pkg/py3.5/linux.txt
index 5e72c45..e919dc1 100644
--- a/requirements/static/pkg/py3.5/linux.txt
+++ b/requirements/static/pkg/py3.5/linux.txt
@@ -16,7 +16,7 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/linux.in
-contextvars==2.4
+contextvars==2.4 ; python_version < "3.7"
     # via -r requirements/base.txt
 cryptography==3.0
     # via pyopenssl
diff --git a/requirements/static/pkg/py3.6/linux.txt b/requirements/static/pkg/py3.6/linux.txt
index f8c5b0e..04bb25e 100644
--- a/requirements/static/pkg/py3.6/linux.txt
+++ b/requirements/static/pkg/py3.6/linux.txt
@@ -16,7 +16,7 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/linux.in
-contextvars==2.4
+contextvars==2.4 ; python_version < "3.7"
     # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
diff --git a/requirements/static/pkg/py3.7/darwin.txt b/requirements/static/pkg/py3.7/darwin.txt
index a69d8ea..111f51f 100644
--- a/requirements/static/pkg/py3.7/darwin.txt
+++ b/requirements/static/pkg/py3.7/darwin.txt
@@ -16,8 +16,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/darwin.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via
     #   -r requirements/darwin.txt
@@ -32,8 +30,6 @@ idna==2.8
     # via
     #   -r requirements/darwin.txt
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/darwin.txt
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.7/freebsd.txt b/requirements/static/pkg/py3.7/freebsd.txt
index 9a6f673..2dfc5ff 100644
--- a/requirements/static/pkg/py3.7/freebsd.txt
+++ b/requirements/static/pkg/py3.7/freebsd.txt
@@ -14,8 +14,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/freebsd.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
@@ -24,8 +22,6 @@ distro==1.5.0
     #   -r requirements/static/pkg/freebsd.in
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/freebsd.in
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.7/linux.txt b/requirements/static/pkg/py3.7/linux.txt
index 99d86d9..9d953a5 100644
--- a/requirements/static/pkg/py3.7/linux.txt
+++ b/requirements/static/pkg/py3.7/linux.txt
@@ -14,16 +14,12 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/linux.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
     # via -r requirements/base.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/linux.in
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.7/windows.txt b/requirements/static/pkg/py3.7/windows.txt
index fd072fd..6d54ac4 100644
--- a/requirements/static/pkg/py3.7/windows.txt
+++ b/requirements/static/pkg/py3.7/windows.txt
@@ -18,8 +18,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/windows.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.4.7
     # via
     #   -r requirements/windows.txt
@@ -32,8 +30,6 @@ gitpython==3.1.18
     # via -r requirements/windows.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/windows.txt
 ioloop==0.1a0
diff --git a/requirements/static/pkg/py3.8/darwin.txt b/requirements/static/pkg/py3.8/darwin.txt
index 648ebd6..9829778 100644
--- a/requirements/static/pkg/py3.8/darwin.txt
+++ b/requirements/static/pkg/py3.8/darwin.txt
@@ -16,8 +16,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/darwin.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via
     #   -r requirements/darwin.txt
@@ -32,8 +30,6 @@ idna==2.8
     # via
     #   -r requirements/darwin.txt
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/darwin.txt
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.8/freebsd.txt b/requirements/static/pkg/py3.8/freebsd.txt
index 9cf85bc..33f3bac 100644
--- a/requirements/static/pkg/py3.8/freebsd.txt
+++ b/requirements/static/pkg/py3.8/freebsd.txt
@@ -14,8 +14,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/freebsd.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
@@ -24,8 +22,6 @@ distro==1.5.0
     #   -r requirements/static/pkg/freebsd.in
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/freebsd.in
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.8/linux.txt b/requirements/static/pkg/py3.8/linux.txt
index 535f7f6..0f7a45e 100644
--- a/requirements/static/pkg/py3.8/linux.txt
+++ b/requirements/static/pkg/py3.8/linux.txt
@@ -14,16 +14,12 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/linux.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
     # via -r requirements/base.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/linux.in
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.8/windows.txt b/requirements/static/pkg/py3.8/windows.txt
index 49415d3..8c3f512 100644
--- a/requirements/static/pkg/py3.8/windows.txt
+++ b/requirements/static/pkg/py3.8/windows.txt
@@ -18,8 +18,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/windows.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.4.7
     # via
     #   -r requirements/windows.txt
@@ -32,8 +30,6 @@ gitpython==3.1.18
     # via -r requirements/windows.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/windows.txt
 ioloop==0.1a0
diff --git a/requirements/static/pkg/py3.9/darwin.txt b/requirements/static/pkg/py3.9/darwin.txt
index c89d860..1b6bca8 100644
--- a/requirements/static/pkg/py3.9/darwin.txt
+++ b/requirements/static/pkg/py3.9/darwin.txt
@@ -16,8 +16,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/darwin.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via
     #   -r requirements/darwin.txt
@@ -32,8 +30,6 @@ idna==2.8
     # via
     #   -r requirements/darwin.txt
     #   requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/darwin.txt
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.9/freebsd.txt b/requirements/static/pkg/py3.9/freebsd.txt
index 1e748a1..6222bde 100644
--- a/requirements/static/pkg/py3.9/freebsd.txt
+++ b/requirements/static/pkg/py3.9/freebsd.txt
@@ -14,8 +14,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/freebsd.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
@@ -24,8 +22,6 @@ distro==1.5.0
     #   -r requirements/static/pkg/freebsd.in
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/freebsd.in
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.9/linux.txt b/requirements/static/pkg/py3.9/linux.txt
index 3df3b14..486d742 100644
--- a/requirements/static/pkg/py3.9/linux.txt
+++ b/requirements/static/pkg/py3.9/linux.txt
@@ -14,16 +14,12 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/static/pkg/linux.in
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.3.2
     # via pyopenssl
 distro==1.5.0
     # via -r requirements/base.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/static/pkg/linux.in
 jaraco.classes==3.2.1
diff --git a/requirements/static/pkg/py3.9/windows.txt b/requirements/static/pkg/py3.9/windows.txt
index b72022b..5b61564 100644
--- a/requirements/static/pkg/py3.9/windows.txt
+++ b/requirements/static/pkg/py3.9/windows.txt
@@ -18,8 +18,6 @@ cheroot==8.5.2
     # via cherrypy
 cherrypy==18.6.1
     # via -r requirements/windows.txt
-contextvars==2.4
-    # via -r requirements/base.txt
 cryptography==3.4.7
     # via
     #   -r requirements/windows.txt
@@ -32,8 +30,6 @@ gitpython==3.1.18
     # via -r requirements/windows.txt
 idna==2.8
     # via requests
-immutables==0.15
-    # via contextvars
 importlib-metadata==4.6.4 ; python_version >= "3.6" and python_version < "3.10"
     # via -r requirements/windows.txt
 ioloop==0.1a0
